profile/README.md

```diff
@@ -1,5 +1,19 @@
+![Flyte Image](https://user-images.githubusercontent.com/27777173/134286401-fd10814d-42f8-4286-b1e0-aa7b8400b1e7.jpg)
 <html>
-<p align="center">
-  Flyte is a <b>workflow automation</b> platform for <b>complex</b>, <b>mission-critical data</b> and <b>ML processes</b> at scale 
-</p>
+  <p align="center">
+   <h3>
+    Flyte is a workflow automation platform for complex, mission-critical data and ML processes at scale 
+   </h3>
+  </p>
 </html>
+
+We at Flyte are constantly striving to make MLOps better by providing off-the-shelf orchestration mechanisms to automate the way you build pipelines. Ours is a **Kubernetes-native** and **type-safe** platform—which we are super proud of! 🥲 
```

Comment:
why the cry emoji

Reply:
Oh, it's not a cry emoji! It's happy tears. Let me change it.